src/components/App/App.tsx

```diff
@@ -50,15 +50,15 @@ function App() {
 
   return (
     <div>
-      {location.pathname !== '/' && <Navbar />}
+      {location.pathname !== '/' && <Navbar currentUser={currentUser}/>}
```

Comment:
Josh recently put in a request that saves the current user to local storage. This way, we can access the current user from anywhere in our app and we don't need to pass currentUser down to child components! We can probably refactor this so we are accessing local storage in `Navbar`. Happy to jump on a call to do this together :)